content/events/one-pager.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Modern Infrastructure as Code
+type: page
+layout: event-one-pager
+url: /show
+noindex: true
+
+meta_desc: Pulumi provides a cloud native programming model for container management. Any code, any cloud, any app.
+
+hero:
+    title: Meet the Pulumi team for tech talks and beer on Wednesday 1/22
+    body: >
+        Join us at Create33 for talks that focus on the development process on K8s.
+        From 5:00pm to 6:00pm we will be hosting a happy hour with the Meetup starting at 6:30pm.
+
+        The happy hour and Meetup are at Create33, which is located on the 33rd floor of the Wells Fargo
+        Center at <a class="underline text-blue-500" href="https://www.google.com/maps/place/Create33/@47.6050632,-122.3339756,15z/data=!4m2!3m1!1s0x0:0xdfce9739a948e539?sa=X&ved=2ahUKEwj5n43yxobnAhWUFzQIHaqaAbYQ_BIwDHoECA0QCA" target="_blank">999 3rd Ave Seattle, WA</a>.
+        The entry doors may be locked when you arrive but we’ll have somebody
+        nearby to let you in. Once inside, please take the elevator to the 33rd floor.
+    code: |
+          import * as kx from "@pulumi/kubernetesx";
+
+          // Define a Pod.
+          const pb = new kx.PodBuilder({ containers: [{ image: "nginx" }] });
+
+          // Create a Kubernetes Deployment using the previous Pod definition.
+          const deployment = new kx.Deployment("nginx", {
+            spec: pb.asDeploymentSpec()
+          });
+
+          // Expose the Deployment using a load balanced Kubernetes Service.
+          const service = deployment.createService({
+              type: kx.types.ServiceType.LoadBalancer,
+          });
+    cta:
+        url: https://www.meetup.com/Seattle-Kubernetes-Meetup/events/267073230/
+        label: Register Here
+
+carousel:
```

Comment:
It seems a shame to duplicate all of this. Could we re-use the existing component somehow?

Reply:
I can't find a way to reuse the YAML definition from the homepage content front matter. We reuse the HTML for it but can't reuse the content definitions. The first solution I can think is to write some sort of script that parses the front matter for something like `includeContent: ["carousel"]`, then rewrite the front matter to include the given content.

Reply:
Zack and I chatted out of band and I'm going to circle back after this and submit a PR that pulls the YAML into the partial, since there's no need to keep them separate. (We did this originally just to avoid git-stomping on one another while finishing up the new home page.)

Reply:
That sounds good to me.

The alternative would be to define the shared values in `config.toml` (it'd have to be converted from YAML to TOML, or switch from `config.toml` to `config.yaml` and use `{{ $.Param "carousel" }}` to get them where used. Individual pages could then override the values in font matter.

But if we don't need to customize these on individual pages, then I'd just move them into the partials themselves, yeah.